src/Symfony/Component/Security/Core/Tests/Authorization/Voter/TraceableVoterTest.php

```diff
@@ -87,8 +87,8 @@ public function testSupportsTypeOnCacheable()
 
     public function testSupportsAttributeOnNonCacheable()
     {
-        $voter = $this->getMockBuilder(VoterInterface::class)->getMockForAbstractClass();
-        $eventDispatcher = $this->getMockBuilder(EventDispatcherInterface::class)->getMockForAbstractClass();
+        $voter = $this->createMock(VoterInterface::class);
+        $eventDispatcher = $this->createStub(EventDispatcherInterface::class);
```

Comment:
Why do we sometimes use `createStub()` and sometimes `createMock()`?

Reply:
I tried to explain my vision here: https://github.com/symfony/symfony/pull/54745#discussion_r1580908218. Open to discussion! 🙂

Reply:
I understand that, but it doesn't seem to be applied consistently if I don't miss anything. Two lines above for example `createStub()` should have also be used for `$voter` then.

Reply:
That's right, I updated to remove this lack of consistency. It should be better now!